apps/web/components/booking/BookingListItem.tsx

```diff
@@ -820,34 +841,32 @@ type NoShowProps = {
   isBookingInPast: boolean;
 };
 
-const Attendee = (attendeeProps: AttendeeProps & NoShowProps) => {
-  const { email, name, bookingUid, isBookingInPast, noShow: noShowAttendee, phoneNumber } = attendeeProps;
+type setAttendeeList = {
+  setAttendeeList: Dispatch<SetStateAction<AttendeeList[]>>;
+};
+
+const Attendee = (attendeeProps: AttendeeProps & NoShowProps & setAttendeeList) => {
+  const { email, name, bookingUid, isBookingInPast, noShow, phoneNumber, setAttendeeList } = attendeeProps;
   const { t } = useLocale();
 
-  const [noShow, setNoShow] = useState(noShowAttendee);
   const [openDropdown, setOpenDropdown] = useState(false);
   const { copyToClipboard, isCopied } = useCopy();
 
   const noShowMutation = trpc.viewer.markNoShow.useMutation({
     onSuccess: async (data) => {
+      const newValue = data.attendees[0];
+      setAttendeeList((old: AttendeeList[]) =>
```

Comment:
```suggestion
      setAttendeeList((oldAttendeeList: AttendeeList[]) =>
```

let's have better naming convention